### ISSUE TITLE
Using in a live environment 

### ISSUE THREAD

**Issue by sword134** [erdogant/caerus#1]
Hello,
I've taken a look at this repo and played around a little, I have yet to understand the exact logic/strategy behind the Caerus. However it needs to be stated that it can only identify stoppoints on the same day one is currently at. Take for example the stock TRV and parse the following parameters into Caerus:

```
start = '2018-01-01'
end = "2020-11-12"

stock_data= yf.download("TRV", start = start, end = end)
cs = caerus(window=50, threshold=0.5, minperc=10, nlargest=1)
cs.fit(stock_data["Adj Close"].values, verbose=3)
cs.plot()
```

As one can see from the plot the stoppoint (red dot) is on the latest/current day (2020-11-12), and the buy point is somewhere around late September. When adjusting the end date to around end of September it doesn't detect/plot the buy point, therefore if one had run Caerus in late September 2020 on "TRV" they would not have gotten a buy signal, only after the stoppoint has been identified will the buy point be marked. Likewise the stoppoint also moves based on the end date. If you change the end date from 2020-11-12 to for example 2020-11-08, then the stop point will move to that date.

I am therefore a bit confused as how to use Caerus in the real world if one was to use this in a trading strategy.

**Comment by bkomaki** [comment]
Me Also have the same result, by increasing  values in trend  it shows a continuous  turning down !
May be we need another indicator to detect long / short signals!

**Comment by erdogant** [comment]
This is correct. It is a lagging indicator. What you can do is identify the known peaks and valleys, extract the features from these points and learn a machine learning model for example.

**Comment by enigma369** [comment]
sadly it's repaint that mean it useless

**Comment by erdogant** [comment]
<img width="1353" height="526" alt="Image" src="https://github.com/user-attachments/assets/60e067ca-e8de-41be-9ba2-6cc3ecf6fc35" />
